op-program/prestates/verify/verify.go

```diff
@@ -68,11 +76,12 @@ func main() {
 		expectedStr := get(expected, i)
```

Comment:
Should the `formatRelease` function used in `get` now also include the type?

Reply:
yes. this should simplify the release comparison a bit. Will do.